pkg/server/image_pull.go

```diff
@@ -199,7 +199,7 @@ func (c *criService) createImageReference(ctx context.Context, name string, desc
 	if err == nil {
 		return nil
 	}
-	if err != nil && !errdefs.IsAlreadyExists(err) {
+	if !errdefs.IsAlreadyExists(err) {
```

Comment:
nit.. can we make this an else case so it's explicit that we meant it so.. :-)

Reply:
it is ok w/o `else` here I think...both are explicit, lol

Reply:
Since last `if` return directly, it is fine not to add an else. :)

Reply:
I suppose if one does not like if else chains a switch is more elegant. But yeah was just a nit :-)